locked-nft/contracts/LockedNFT.cdc

```diff
@@ -0,0 +1,131 @@
+import NonFungibleToken from "./NonFungibleToken.cdc"
+
+pub contract LockedNFT {
+
+    pub event ContractInitialized()
+    pub event NFTLocked(
```

Comment:
probably can add an NFT type to this event

Reply:
Yes. Will do, plus unlockedOn. Thanks Eric.

Reply:
@HouseOfHufflepuff We would need lockedOn and lockedUntil in the event to track it in our end.

Reply:
Done.